src/ecs/resources/input.rs

```diff
@@ -45,7 +45,7 @@ impl InputHandler {
                             // second `Pressed` event.
                         }
                         Entry::Vacant(entry) => {
-                            entry.insert(KeyQueryState::Queried);
+                            entry.insert(KeyQueryState::NotQueried);
```

Comment:
My reasoning here is: when the key is pressed it has not been queried, so key_once (etc.) will return true.